src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_SIZE_VALUE = 6;
```

Comment:
```suggestion
    private static final int MAX_SIZE_VALUE = 6;
```
constants must have a private access modifier